templates/graphdb/statefulset.yaml

```diff
@@ -126,6 +126,60 @@ spec:
           secret:
             secretName: {{ (tpl .Values.security.initialUsers.existingSecret .) | default (include "graphdb.fullname.secret.initial-users" .) }}
         {{- end }}
+        {{- with .Values.configuration.tls }}
+        {{- if .keystore.secretName}}
+        - name: graphdb-connector-keystore
+          secret:
+            secretName: {{ .keystore.secretName }}
+        {{- end }}
+        {{- if .truststore.secretName}}
+        - name: graphdb-connector-truststore
+          secret:
+            secretName: {{ .truststore.secretName }}
+        {{- end }}
+        {{- if .certificateRevocationList.secretName }}
+        - name: graphdb-connector-certificate-revocation-list
+          secret:
+            secretName: {{ .certificateRevocationList.secretName }}
+        {{- end }}
+        {{- end }}
+        {{- with .Values.cluster.tls }}
+        {{- if .keystore.secretName}}
+        - name: graphdb-keystore
+          secret:
+            secretName: {{ .keystore.secretName }}
+        {{- end }}
+        {{- if .truststore.secretName}}
+        - name: graphdb-truststore
+          secret:
+            secretName: {{ .truststore.secretName }}
+        {{- end }}
+        {{- if .privateKey.secretName }}
+        - name: graphdb-private-key
```

Comment:
Although the documentation says otherwise, the connector could possibly be configured using certificate and private key. Needs to be tested in the future.